htdocs/core/class/commonobject.class.php

```diff
@@ -8258,7 +8261,7 @@ public function setVarsFromFetchObj(&$obj)
 	 * @param   string   $alias   	String of alias of table for fields. For example 't'.
 	 * @return  string				list of alias fields
 	 */
-	protected function getFieldList($alias = '')
+	public function getFieldList($alias = '')
```

Comment:
Why do you need "public" instead of "protected" ?

Reply:
To allow getfiled list from list files to build sql rather than "foreach"